### DISCUSSION TITLE
Does Amazon Recieve/ Train on User Data Used with Chronos T5 Small?

### DISCUSSION THREAD

**Discussion by smartaces** [amazon-science/chronos-forecasting#116] [Q&A] [answered]
So, I'll admit I'm a bit new to all of this. 

I'm used to building around LLM API calls to OpenAI and Anthropic. 

I created  a Google Colab Notebook, which allows me to run and test Amazon Chronos on small datasets:

https://colab.research.google.com/drive/1alfoBOkoi7qponuD5mw9FX0QMHZNqMB9#scrollTo=MRRy4l-HtfJC

It starts of with installing the github package:

!pip install git+https://github.com/amazon-science/chronos-forecasting.git

My question is, and sorry if I am being really dumb...

When I run forecasts against some data (so far I am only using Kaggle sets), is the data being sent anywhere... as in would data be retained anywhere (by Amazon) for further model training. 

If someone were to test the colab notebook, using their real sales data, would that data be used by Amazon for model training. 

Or in the instance of the Colab notebook, is T5 small being installed on a Google virtual machine (effectively) so the data isn't being sent anywhere else. 

So fundamentally, when using Chronos T5 Small (or any other) is that data sent to, or retained by Amazon for further model training.

Many thanks

J

**Comment by abdulfatir** [comment, marked as answer]
@smartaces Thanks for your question. Chronos models are publicly available models that you run _locally_ (on your machine or colab). Therefore, there is no API or remote call involved and all data stays on your machine.

**Reply by smartaces** [reply]
Dude... thanks so much - this has made my day. 

All the best

J
